### PR TITLE
[Home][ArtworkRail] Routing for header

### PR DESCRIPTION
**Pull request by sarahscott**
Routing for existing contexts:
- [x] **Works by artists you follow** => Works For You vc
- [x] **Gene** => Gene vc
- [x] **Fair** => Fair vc
- [x] **Auction** => Auction vc
- [x] **Works by [Artist] based on [OtherArtist]** => Artist page of [Artist]
- [x] Follow/Unfollow related artist

### CODE REVIEW

File: lib/components/home/artwork_rails/artwork_rail_header.js [hunk 1]
```diff
@@ -3,25 +3,51 @@
...
 import Relay from 'react-relay'
 import React from 'react'
-import { StyleSheet, Dimensions, View, Text } from 'react-native'
+import { StyleSheet, Dimensions, View, Text, TouchableWithoutFeedback } from 'react-native'
 
 import SerifText from '../../text/serif'
 import colors from '../../../../data/colors'
 import Button from '../../buttons/inverted_button'
+import SwitchBoard from '../../../modules/switch_board'
+import fragments from './relay_fragments'
```
Comment (sarahscott) [inline lib/components/home/artwork_rails/artwork_rail_header.js]: These were cluttering up this component quite a bit so I moved them into their own file. A lot of the routing here (and therefore these fragments) is identical to the routing for the View All button in the main `ArtworkRail` component. I'm currently thinking about nice ways to abstract the routing and fragments so they can be used by both components.

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/e20eaf4aac7960e7aa42c2d40a09ae860e7696b3/lib/components/home/artwork_rails/artwork_rail_header.js">lib/components/home/artwork_rails/artwork_rail_header.js</a> &amp; <a href="https://github.com/artsy/emission/blob/e20eaf4aac7960e7aa42c2d40a09ae860e7696b3/lib/components/home/artwork_rails/relay_fragments.js">lib/components/home/artwork_rails/relay_fragments.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: lib/components/home/artwork_rails/artwork_rail_header.js (hunk 1 above)
Comment (sarahscott) [inline lib/components/home/artwork_rails/artwork_rail_header.js]: Maybe I'll move `Switchboard` into a file with the fragments and just have methods for routing and relay. Like a network model.

**Comment by sarahscott** [comment]
I'm happy with this until (or unless) the team agrees on a different routing strategy.

**Comment by orta** [comment]
Cool, looks like just tests then you can probably cover some of the routing inside `handleFollowChange` and `handleViewAll` 👍

**Comment by orta** [comment]
and a CHANGELOG entry 💃

**Comment by mennenia** [comment]
Awesome. This fixes [this issue](https://github.com/artsy/eigen/issues/1761), so I've assigned you to that just so you can close it because 🎉 it's done!

**Comment by alloy** [comment]
As discussed in Slack, skip the tests for now, but do add a CHANGELOG entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ### Master
 
+- Added an Artworks rail to Home view - sarah
```
Comment (sarahscott) [inline CHANGELOG.md]: 🎉

**Comment by orta** [comment]
👍

**Pull request merged**